javascript/exercises/validateCreditCardSolution.js

```diff
@@ -7,24 +7,16 @@ The sum of all the digits must be greater than 16
 */
 
 function validateCreditCard(creditCardNum){
-  
```

Comment:
Can we keep the code for removing dashes as a separate solution file for the bonus?